README.md

```diff
@@ -29,6 +29,11 @@ sudo install -Dm644 zsh/_ytcc /usr/share/zsh/site-functions/_ytcc
 yaourt -S ytcc
 ```
 
+### Void Linux
+```shell
+sudo xbps-install -S ytcc
```

Comment:
Please remove the -S. It is not necessary.

Reply:
@Vaelatern -S simply syncs the repository before fetching the package. While not strictly necessary it ensures the installation is successful and of the latest version.

For more info: https://wiki.voidlinux.eu/XBPS#xbps-install

Reply:
Yes.
But this should be known by Void users. There are a lot of guides encouraging an understanding of `xbps` commands as practically pacman. Considering that people learn from random snippets, I'd rather fewer snippets with `-S` without cause.

Reply:
I'll leave it in, because the project seems to endorse using `-S`. In the Wiki article on xbps there is not a single `xbps-install` without `-S`.